runtime/gc_glue_java/configure_includes/configure_aix_ppc.mk

```diff
@@ -64,6 +64,16 @@ ifeq (default,$(origin CXX))
 	CXX = $(CC)
 endif
 
+ifneq (,$(findstring ibm-clang,$(notdir $(CC))))
+    CONFIGURE_ARGS += 'OMR_TOOLCHAIN=openxl'
+    CONFIGURE_ARGS += 'GLOBAL_CFLAGS=-fstack-protector'
+    CONFIGURE_ARGS += 'GLOBAL_CXXFLAGS=-fstack-protector'
+else
+    CONFIGURE_ARGS += 'OMR_TOOLCHAIN=xlc'
+    CONFIGURE_ARGS += 'GLOBAL_CFLAGS=-qstackprotect'
+    CONFIGURE_ARGS += 'GLOBAL_CXXFLAGS=-qstackprotect'
+endif
```

Comment:
Suggest this should remain where it was (after line 88).
Please don't remove the final newline from text files.